src/Microsoft.Data.SqlClient/tests/ManualTests/ProviderAgnostic/ReaderTest/ReaderTest.cs

```diff
@@ -276,84 +276,86 @@ public static void SqlDataReader_SqlBuffer_GetFieldValue()
         {
             string tableName = DataTestUtility.GetUniqueNameForSqlServer("SqlBuffer_GetFieldValue");
             DateTimeOffset dtoffset = DateTimeOffset.Now;
-            DateTime dateTime = DateTime.Now;
+            DateTime dt = DateTime.Now;
+            //Exclude the millisecond because of rounding at some points by SQL Server.
+            DateTime dateTime = new DateTime(dt.Year, dt.Month, dt.Day, dt.Hour, dt.Minute, dt.Second);
```

Comment:
couldn't we use an overload of ToString? something like `ToString("hh:mm:ss")`

Reply:
Does this have any issue that will fix with your suggestion?

Reply:
No, it will just make it easier to follow the test pattern.